### ISSUE TITLE
Scope to packages inside a conda environment

### ISSUE THREAD

**Issue by jameslamb** [raimon49/pip-licenses#26]
@raimon49 thanks for this awesome project!

I'd like to request a feature...could you make it possible to exclude globally-visible packages if you are inside a conda environment?

For my use case, I'd like to use `pip-licenses` in CI to catch dependencies with problematic licenses. However, right now I have to manually exclude a lot of stuff because things installed in my local site-packages are making it through.

Basically, I'm asking if `pip-licenses` could replicate whatever `pip freeze --local` does.

**Comment by raimon49** [comment]
@jameslamb Thank you for the feature request.

I currently don't use the conda environment. However, if I can reproduce your conda environment, I may be able to implement this feature request.

Please comment on the information to reproduce your conda environment.

* Python version
* Anaconda version
* And more

**Comment by jameslamb** [comment]
Going through old issues, I realize I never responded here. It's been 6 years since I opened this so I don't remember the original context.

Let's close this, sorry for losing track of this one.
